--- dvdisaster-0.66/GNUmakefile.template.orig	2006-03-21 20:04:19.000000000 +0100
+++ dvdisaster-0.66/GNUmakefile.template	2006-06-19 04:54:22.593421750 +0200
@@ -181,26 +181,6 @@
 	@echo "Installing package..."
 	install -d $(BUILDROOT)$(BINDIR)
 	install dvdisaster$(EXE_SUFFIX) $(BUILDROOT)$(BINDIR)
-	install -d $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 CHANGELOG $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 COPYING $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 CREDITS* $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 TODO $(BUILDROOT)$(DOCSUBDIR)
-	install -d $(BUILDROOT)$(DOCSUBDIR)/cs
-	cd documentation; install -m 644 cs/*.html $(BUILDROOT)$(DOCSUBDIR)/cs
-	install -d $(BUILDROOT)$(DOCSUBDIR)/de
-	cd documentation; install -m 644 de/*.html $(BUILDROOT)$(DOCSUBDIR)/de
-	install -d $(BUILDROOT)$(DOCSUBDIR)/en
-	cd documentation; install -m 644 en/*.html $(BUILDROOT)$(DOCSUBDIR)/en
-	install -d $(BUILDROOT)$(DOCSUBDIR)/images
-	cd documentation; install -m 644 images/*.png $(BUILDROOT)$(DOCSUBDIR)/images
-	cd documentation; install -m 644 images/*.jpg $(BUILDROOT)$(DOCSUBDIR)/images
-	install -d $(BUILDROOT)$(DOCSUBDIR)/cs/images
-	cd documentation/cs; install -m 644 images/*.png $(BUILDROOT)$(DOCSUBDIR)/cs/images
-	install -d $(BUILDROOT)$(DOCSUBDIR)/de/images
-	cd documentation/de; install -m 644 images/*.png $(BUILDROOT)$(DOCSUBDIR)/de/images
-	install -d $(BUILDROOT)$(DOCSUBDIR)/en/images
-	cd documentation/en; install -m 644 images/*.png $(BUILDROOT)$(DOCSUBDIR)/en/images
 	install -d $(BUILDROOT)$(MANDIR)/man1
 	install -d $(BUILDROOT)$(MANDIR)/cs/man1
 	install -d $(BUILDROOT)$(MANDIR)/de/man1
@@ -223,17 +203,6 @@
 	    install -m 644 $$i/LC_MESSAGES/dvdisaster.mo $(BUILDROOT)$(LOCALEDIR)/$$i/LC_MESSAGES; \
 	  done \
 	fi
-	@echo "# dvdisaster uninstaller script" > $(BINDIR)/dvdisaster-uninstall.sh
-	@echo "# Usage: sh dvdisaster-uninstall.sh" >> $(BINDIR)/dvdisaster-uninstall.sh
-	@echo >> $(BINDIR)/dvdisaster-uninstall.sh
-	@echo "rm -f $(BINDIR)/dvdisaster $(BINDIR)/dvdisaster.exe $(BINDIR)/dvdisaster-win.exe $(BINDIR)/dvdisaster-uninstall.sh" >> $(BINDIR)/dvdisaster-uninstall.sh
-	@echo "rm -rf $(DOCSUBDIR)" >> $(BINDIR)/dvdisaster-uninstall.sh
-	@echo "rm -f $(MANDIR)/man1/dvdisaster.1" >> $(BINDIR)/dvdisaster-uninstall.sh
-	@for i in cs de it; do \
-	  echo "rm $(LOCALEDIR)/$$i/LC_MESSAGES/dvdisaster.mo" >> $(BINDIR)/dvdisaster-uninstall.sh ; \
-	  echo "rm $(MANDIR)/$$i/man1/dvdisaster.1" >> $(BINDIR)/dvdisaster-uninstall.sh ; \
-	done
-	@echo "echo \"dvdisaster has been uninstalled.\"" >> $(BINDIR)/dvdisaster-uninstall.sh
 	
 uninstall:
 	@echo "Uninstalling package..."
