--- dvdisaster-0.62/Makefile.template.orig	2005-06-27 15:20:24.000000000 +0200
+++ dvdisaster-0.62/Makefile.template	2005-07-03 22:21:10.000000000 +0200
@@ -119,23 +119,23 @@
 install: dvdisaster
 	@if ! test -e documentation/de/index.html; then make html; fi 
 	@echo "Installing package..."
-	install -d $(BINDIR)
-	install dvdisaster$(EXE_SUFFIX) $(BINDIR)
-	install -d $(DOCDIR)
-	install COPYING $(DOCDIR)
-	install CHANGELOG $(DOCDIR)
-	install TODO $(DOCDIR)
-	install -d $(DOCDIR)/de
-	cd documentation; install -m 644 de/* $(DOCDIR)/de
-	install -d $(DOCDIR)/en
-	cd documentation; install -m 644 en/* $(DOCDIR)/en
-	install -d $(DOCDIR)/images
-	cd documentation; install -m 644 images/* $(DOCDIR)/images
+	install -d $(DESTDIR)$(BINDIR)
+	install dvdisaster$(EXE_SUFFIX) $(DESTDIR)$(BINDIR)
+	install -d $(DESTDIR)$(DOCDIR)
+	install COPYING $(DESTDIR)$(DOCDIR)
+	install CHANGELOG $(DESTDIR)$(DOCDIR)
+	install TODO $(DESTDIR)$(DOCDIR)
+	install -d $(DESTDIR)$(DOCDIR)/de
+	cd documentation; install -m 644 de/* $(DESTDIR)$(DOCDIR)/de
+	install -d $(DESTDIR)$(DOCDIR)/en
+	cd documentation; install -m 644 en/* $(DESTDIR)$(DOCDIR)/en
+	install -d $(DESTDIR)$(DOCDIR)/images
+	cd documentation; install -m 644 images/* $(DESTDIR)$(DOCDIR)/images
 	if echo $(WITH_OPTIONS) | grep -q "NLS_YES"; then \
 	  cd locale; \
 	  for i in ??; do\
-	    install -d $(LOCALEDIR)/$$i/LC_MESSAGES; \
-	    install $$i/LC_MESSAGES/dvdisaster.mo $(LOCALEDIR)/$$i/LC_MESSAGES; \
+	    install -d $(DESTDIR)$(LOCALEDIR)/$$i/LC_MESSAGES; \
+	    install $$i/LC_MESSAGES/dvdisaster.mo $(DESTDIR)$(LOCALEDIR)/$$i/LC_MESSAGES; \
 	  done \
 	fi
 
