--- GNUmakefile.template	2006-07-09 14:35:01.000000000 +0200
+++ GNUmakefile.template-new	2006-07-11 17:28:03.545563376 +0200
@@ -210,27 +210,6 @@
 	@echo "Installing package..."
 	install -d $(BUILDROOT)$(BINDIR)
 	install dvdisaster$(EXE_SUFFIX) $(BUILDROOT)$(BINDIR)
-	install -d $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 CHANGELOG $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 COPYING $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 CREDITS* $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 README.MODIFYING $(BUILDROOT)$(DOCSUBDIR)
-	install -m 644 TODO $(BUILDROOT)$(DOCSUBDIR)
-	install -d $(BUILDROOT)$(DOCSUBDIR)/cs
-	cd documentation; install -m 644 cs/*.html $(BUILDROOT)$(DOCSUBDIR)/cs
-	install -d $(BUILDROOT)$(DOCSUBDIR)/de
-	cd documentation; install -m 644 de/*.html $(BUILDROOT)$(DOCSUBDIR)/de
-	install -d $(BUILDROOT)$(DOCSUBDIR)/en
-	cd documentation; install -m 644 en/*.html $(BUILDROOT)$(DOCSUBDIR)/en
-	install -d $(BUILDROOT)$(DOCSUBDIR)/images
-	cd documentation; install -m 644 images/*.png $(BUILDROOT)$(DOCSUBDIR)/images
-	cd documentation; install -m 644 images/*.jpg $(BUILDROOT)$(DOCSUBDIR)/images
-	install -d $(BUILDROOT)$(DOCSUBDIR)/cs/images
-	cd documentation/cs; install -m 644 images/*.png $(BUILDROOT)$(DOCSUBDIR)/cs/images
-	install -d $(BUILDROOT)$(DOCSUBDIR)/de/images
-	cd documentation/de; install -m 644 images/*.png $(BUILDROOT)$(DOCSUBDIR)/de/images
-	install -d $(BUILDROOT)$(DOCSUBDIR)/en/images
-	cd documentation/en; install -m 644 images/*.png $(BUILDROOT)$(DOCSUBDIR)/en/images
 	install -d $(BUILDROOT)$(MANDIR)/man1
 	install -d $(BUILDROOT)$(MANDIR)/cs/man1
 	install -d $(BUILDROOT)$(MANDIR)/de/man1
@@ -253,19 +232,6 @@
 	    install -m 644 $$i/LC_MESSAGES/dvdisaster.mo $(BUILDROOT)$(LOCALEDIR)/$$i/LC_MESSAGES; \
 	  done \
 	fi
-	@echo "# dvdisaster uninstaller script" > $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh
-	@echo "# Usage: sh dvdisaster-uninstall.sh" >> $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh
-	@echo >> $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh
-	@echo "rm -f $(BINDIR)/dvdisaster $(BINDIR)/dvdisaster.exe $(BINDIR)/dvdisaster-win.exe $(BINDIR)/dvdisaster-uninstall.sh" >> $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh
-	@echo "rm -rf $(DOCSUBDIR)" >> $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh
-	@echo "rm -f $(MANDIR)/man1/dvdisaster.1" >> $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh
-	@for i in $(PO_LOCALES); do \
-	  echo "rm $(LOCALEDIR)/$$i/LC_MESSAGES/dvdisaster.mo" >> $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh ; \
-	done
-	@for i in $(MAN_LOCALES); do \
-	  echo "rm $(MANDIR)/$$i/man1/dvdisaster.1" >> $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh ; \
-	done
-	@echo "echo \"dvdisaster has been uninstalled.\"" >> $(BUILDROOT)$(BINDIR)/dvdisaster-uninstall.sh
 	
 uninstall:
 	@echo "Uninstalling package..."
@@ -410,4 +376,3 @@
 zipdist: prepare-dist
 	@echo "Packaging with zip"
 	@echo "MS Windows standalone distribution of $(PKGNAME)." | zip -S9rqoz $(PKGNAME).zip $(PKGNAME)
-
